specification/datamigration/resource-manager/Microsoft.DataMigration/stable/2018-04-19/datamigration.json

```diff
@@ -0,0 +1,1239 @@
+        "tags": [
+          "Task resource",
+          "Standard operation",
+          "PATCH"
+        ],
+        "operationId": "Tasks_Update",
+        "x-ms-examples": {
+          "Tasks_Update": { "$ref": "./examples/Tasks_Update.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/projectTask"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Task resource updated",
+            "schema": {
+              "$ref": "./definitions/Tasks.json#/definitions/ProjectTask"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "./definitions/Common.json#/definitions/ApiError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{groupName}/providers/Microsoft.DataMigration/services/{serviceName}/projects/{projectName}/tasks/{taskName}/cancel": {
+      "post": {
+        "summary": "Cancel a task",
+        "description": "The tasks resource is a nested, proxy-only resource representing work performed by a DMS instance. This method cancels a task if it's currently queued or running.",
+        "tags": [
+          "Task resource",
+          "Custom operation",
+          "POST"
+        ],
+        "operationId": "Tasks_Cancel",
+        "x-ms-examples": {
+          "Tasks_Cancel": { "$ref": "./examples/Tasks_Cancel.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/groupName"
+          },
+          {
+            "$ref": "#/parameters/serviceName"
+          },
+          {
+            "$ref": "#/parameters/projectName"
+          },
+          {
+            "$ref": "#/parameters/taskName"
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Task canceled",
+            "schema": {
+              "$ref": "./definitions/Tasks.json#/definitions/ProjectTask"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "./definitions/Common.json#/definitions/ApiError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{groupName}/providers/Microsoft.DataMigration/services/{serviceName}/projects": {
+      "get": {
+        "summary": "Get projects in a service",
+        "description": "The project resource is a nested resource representing a stored migration project. This method returns a list of projects owned by a service resource.",
+        "tags": [
+          "Project resource",
+          "Standard operation",
+          "GET"
+        ],
+        "operationId": "Projects_List",
+        "x-ms-examples": {
```

Comment:
Operation ID should be "Projects_ListByResourceGroup"

Reply:
Done.